src/test/java/com/limechain/runtime/hostapi/MiscellaneousHostFunctionsTest.java

```diff
@@ -66,7 +66,7 @@ void printHexV1() {
     }
 
     @Test
-    @Disabled("Does not work in github actions")
+    @Disabled("Failing probably because of runtime refactor.")
```

Comment:
Is fixed after the merge of #464 in 4f388cb7e2cc0434189d813fc4c65c30052457a9. Merge or rebase on `dev` and try again. 😄